apps/website/src/pages/docs/tile.mdx

```diff
@@ -14,14 +14,26 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tile' />
+{/* <Placeholder componentPageName='core-tile' /> */}
```

Comment:
Remove if not needed anymore or add back if you're not ready to update docs page just yet.

Reply:
Oops, actually not meant to push this file, I have reverted the changes.